test/unit/test_segment_tree.cpp

```diff
@@ -353,13 +353,12 @@ TEST(segment_tree, tag_roots) {
             EXPECT_EQ((std::vector<arb::msize_t>{3}), arb::tag_roots(tree, 3));
         }
     }
-    /* gnarly tree
-    ** (npos) - 0 - 1 - 4
-    **            \
-    **              2 - 3
-    **                \
-    **                  5
-    */
+    // gnarly tree
+    // (npos) - 0 - 1 - 4
+    //            \
+    //              2 - 3
+    //                \
+    //                  5
```

Comment:
please use master version (this is re-introducing stuff that was merged into master if I am not mistaken)

Reply:
I have no clue why this keeps on coming back.